src/expression/comparison/__test__/unit/eq.test.ts

```diff
@@ -1,30 +1,37 @@
-import { Result } from '../../../../common/evaluable'
+import {
```

Comment:
Shouldn't strict/not strict comparison cases both be tested for `eq` as well?

Reply:
💯 ... And this https://github.com/briza-insurance/illogical/pull/137#discussion_r574506004 I guess be captured.